packages/beacon-node/test/unit-mainnet/network/gossip/scoringParameters.test.ts

```diff
@@ -80,7 +80,7 @@ describe("computeGossipPeerScoreParams", function () {
     expect(params.firstMessageDeliveriesWeight).closeTo(1.8407, TOLERANCE);
     expect(params.firstMessageDeliveriesDecay).closeTo(0.99856, TOLERANCE);
     expect(params.firstMessageDeliveriesCap).closeTo(21.73035, TOLERANCE);
-    expect(params.invalidMessageDeliveriesWeight).closeTo(-2150, TOLERANCE);
+    expect(params.invalidMessageDeliveriesWeight).closeTo(-2200, TOLERANCE);
```

Comment:
@dapplion  I am assuming the weights changed because of addition of new topic, in case that assumption is incorrect, will dig deeper into it

Reply:
CC: @tuyennhv

Reply:
`expectedMessageRate` is quite tricky but we can tweak this later. Looking good for now